src/layouts/AuthorizationWrapper.tsx

```diff
@@ -2,6 +2,7 @@ import { useRouter } from "next/router";
 import React, { Fragment } from "react";
 import ErrorScreen, { ErrorScreenType } from "../components/ErrorScreen";
 import { AuthorizationLevel, useAuthorizationLevel } from "../hooks";
+import AuthModal from "../components/Authentication/AuthModal";
```

Comment:
nit: I'd call this `AuthenticationModal` to avoid potential confusion with "Authorization"